R/human_infection.R

```diff
@@ -229,6 +249,53 @@ infection_outcome_process <- function(
   )
 }
 
+#' @title Calculate patent infections (p.v only)
+#' @description
+#' Sample patent infections from all infections
+#' @param variables a list of all of the model variables
+#' @param infections bitset of infected humans
+#' @param parameters model parameters
+#' @param renderer model render
+#' @param timestep current timestep
+#' @noRd
+calculate_patent_infections <- function(
+    variables,
+    infections,
+    parameters,
+    renderer,
+    timestep
+) {
+  
+  iaa <- variables$iaa$get_values(infections)
+  iam <- variables$iam$get_values(infections)
+  
+  philm <- anti_parasite_immunity(
+    min = parameters$philm_min, max = parameters$philm_max, a50 = parameters$alm50,
+    k = parameters$klm, iaa = iaa, iam = iam)
+  patent_infections <- bitset_at(infections, bernoulli_multi_p(philm))
+  
+  incidence_renderer(
```

Comment:
Might be worth chatting through this in person - I'm not clear what `patent_incidence` is really and how it would be used.